src/util/GeoSparqlHelpers.cpp

```diff
@@ -0,0 +1,5 @@
+// Copyright 2021, University of Freiburg,
+// Author: Hannah Bast <bast@cs.uni-freiburg.de>
+// Chair of Algorithms and Data Structures
+
+#include "./GeoSparqlHelpers.h"
```

Comment:
This file currently is empty.

Reply:
It was there and empty already before this PR, but now it's no longer empty :-)